src/index.js

```diff
@@ -50,6 +54,8 @@ export function parseValue(value) {
     return parseList(value);
   } else if (_.isPlainObject(value)) {
     return parseMap(value);
+  } else if (ESCAPE_STRINGS && _.isString(value)) {
```

Comment:
Ah, while updating my PR I've realized why I've used the variable as intermediate: `this` is undefined in `parseValue`.